include/koa/authentication/passport-strategies/salesforce_authentication_strategy.js

```diff
@@ -24,9 +25,9 @@ module.exports = (pb) => {
         pluginService = Promise.promisifyAll(pluginService);
         const settings = await pluginService.getSettingsKVAsync('tn_join');
```

Comment:
Remember, we need to change this from tn_join before this goes live.

Reply:
Yes, I've already created the card for this. Going to do it when we deploy cms-koa branch to master